Description: fix ffmpeg builds on x86, where there aren't enough registers
Origin: https://chromium.googlesource.com/chromium/third_party/ffmpeg.git/+/5dccf033d1d44afeb2901fb6239268eba9ce39de

--- a/third_party/ffmpeg/BUILD.gn
+++ b/third_party/ffmpeg/BUILD.gn
@@ -241,11 +241,9 @@ target(link_target_type, "ffmpeg_interna
   # Windows builds can't compile without EBP because we can't omit frame
   # pointers like we do on posix.
   if (target_cpu == "x86") {
-    if (using_sanitizer || is_win) {
       defines += [ "HAVE_EBP_AVAILABLE=0" ]
     } else {
       defines += [ "HAVE_EBP_AVAILABLE=1" ]
-    }
   }
 
   if (is_posix || is_fuchsia) {
